crate/pages/blog/release-v0-46-4.html

```diff
@@ -0,0 +1,119 @@
+title: New release: v0.46.4
+link: https://crate.io/blog/release-v0-46-4/
+author: Kenan
+description: Crate Release v0.46.4
+created: 2015/02/06 14:00
+post_name: new release: v0.46.4
+status: publish
+post_type: post
+tags: crate, release, elasticsearch
+category: developernews
+
+Crate.IO is pleased to announce a new release of our distributed database for Docker. You can find a summary of changes in the [CHANGES.txt](https://github.com/crate/crate/blob/master/CHANGES.txt) file in our GitHub repository. In this revision we are announcing a few new features, we are strive to make updating to our latest releases seamless and straighforward. If you find this to not be the case, please do not hesitate to [contact us](https://crate.io/about/contact/) and let us know.
```

Comment:
> "Crate.IO is pleased to announce a new release of our distributed database for Docker"

We're pleased to announce that Crate 0.46.x is now stable.

> "In this revision we are announcing a few new features, we are strive to make updating to our latest releases seamless and straightforward ..."

sounds very ... (don't know how to describe it) to me. and version ≠ revision